src/slider/slider.stories.ts

```diff
@@ -31,6 +31,7 @@ storiesOf("Components|Slider", module).addDecorator(moduleMetadata({
 			[value]="value"
 			[shiftMultiplier]="shiftMultiplier"
 			[disabled]="disabled"
+			aria-Label="Label for slider value"
```

Comment:
Just double checking that this should be here an not an input?

Reply:
It should be for the input, not sure why I changed what it was originally!

Reply:
It should be for the input, not sure why I changed what it was originally!